### ISSUE TITLE
No infrared signals with delay possible.

### ISSUE THREAD

**Issue by Gand41f** [athombv/homey#719]
**What is your Homey version (Settings → System)?**

0.8.39

**What did you try to achieve?**

Create a flow, which does send infrared signals with and without a delay.

**What was the result?**

Homey does only send the infrared signals, which do not have a delay. Infrared signals with a delay are not beeing send.

**What did you expect as result?**

Homey should send all infrared signals independent on whether they do have a delay or not.

**Any other remarks**
Problem was actually related to the infrared range, which seems to be about 1.5m in my case (see below).

**Comment by bvdbos** [comment]
see #437 -> this is (or at least was...) only the case when using the test-button but it worked as expected when using other triggers...

**Comment by Gand41f** [comment]
I do use a button on a 433 MHz remote from Smartwares as a trigger. I does not work with this one as well as with the test button.

**Comment by Gand41f** [comment]
I did investigate the problem a bit closer. It seems to be a problem of the infrared range. I checked with the camera of my phone whether Homey was actually emitting something and it did every time as it was supposed to.
Homey was placed about 3m away from the video projector which should be switched on with infrared. Here it only worked sporadically, which it orignally accounted to the time delay. When placing homey 1.5m away from the projector it does work very reliably. Wasn't homey supposed to have a very large infrared range of ~10m?
